src/Microsoft.PowerShell.Commands.Management/commands/management/Service.cs

```diff
@@ -1541,15 +1541,16 @@ public string Description
 
         internal string description = null;
 
-        /// <summary>
-        /// The following is the definition of the input parameter "StartupType".
-        /// Changes the starting mode of the service. Valid values for StartupType are:
-        /// -- Automatic: Start when the system starts.
-        /// -- Manual   : Starts only when started by a user or program.
-        /// -- Disabled : Can.
-        /// </summary>
-        [Parameter]
-        [Alias("StartMode", "SM", "ST")]
+		/// <summary>
+		/// The following is the definition of the input parameter "StartupType".
+		/// "StartType" is NativeMethod Get;Set; ServiceController.InputObject.StartType
+		/// Changes the starting mode of the service. Valid values for StartupType are:
+		/// -- Automatic: Start when the system starts.
+		/// -- Manual   : Starts only when started by a user or program.
+		/// -- Disabled : Can.
+		/// </summary>
+		[Parameter]
+        [Alias("StartMode", "SM", "ST", "StartType")]
```

Comment:
Why do you change indentation above? Please revert.

Also we could discuss to swap StartupType and StartType names.

Reply:
I think visualstudio is auto adding my c# projects... I usually work in C++

Reply:
Our repo has editor configuration settings - please use them.